CHANGES_NEXT_RELEASE

```diff
@@ -1,4 +1,5 @@
-ADD PM2_ENABLED flag to Docker
-Upgrade NodeJS version from 8.16.0 to 8.16.1 in Dockerfile due to security issues
-
-NOTE: in the "upgrade iotagent-node-lib from X to Y" include a mention about NGSIv2 forwarding support and issue #250
+- ADD PM2_ENABLED flag to Docker.
+- Upgrade NodeJS version from 8.16.0 to 8.16.1 in Dockerfile due to security issues.
+- Configure multithreading using cluster nodejs module (through multicore config.js parameter or IOTA_MULTI_CORE
```

Comment:
```suggestion
- NOTE: in the "upgrade iotagent-node-lib from X to Y" include a mention to configure multithreading using cluster nodejs module (through multicore config.js parameter or IOTA_MULTI_CORE)
```

This is due to the actual change is not in the agents code. So, the changelog in this case is a bit different. Don't worry about that, we will deal with it at release time :)

Reply:
Fixed in e75739a